### ISSUE TITLE
Heuristics-Procedural 1d

### ISSUE THREAD

**Issue by tiffanyrzhou** [REUL-Lab/eula-aat#14]
Procedural 1d - Ensure accessibility of EULA on mobile devices: The tool will check if the EULA does not have a clear text size and contrast. The tool will assign a grade F if the text size is less than 12 or if the text does not contrast with the background, and an A if font size is 12 or greater and the text contrasts with the background.
